src/elements/content-sidebar/Sidebar.js

```diff
@@ -288,4 +381,4 @@ class Sidebar extends React.Component<Props, State> {
 }
 
 export { Sidebar as SidebarComponent };
-export default flow([withFeatureConsumer, withRouter])(Sidebar);
+export default flow([withAPIContext, withFeatureConsumer, withRouter])(Sidebar);
```

Comment:
Minor, but if we're adding a HOC here we could create a `withCurrentUser` HOC instead to encapsulate the lazy-loading logic.